### ISSUE TITLE
Озвучить текст

### ISSUE THREAD

**Issue by andrei-podholms** [minsk-hackerspace/alisa#8]
Есть датчик который может предупредить алису о опасности.
Хочется чтобы она могла предупредить ХСовцев.
Как это сделать?

А точнее есть ли в ХС демон RHVoice на который можно послать текст для проговаривания?

**Comment by abitrolly** [comment]
Тебе нужен @sergejey. Напиши инструкцию как узнаешь.

**Comment by sergejey** [comment]
А что за датчик? Может имеет смысл просто его показания или событие в Алису кидать (http, mqtt и так далее), а там уже прописать логику, что с этим делать.

**Comment by abitrolly** [comment]
@sergejey а можно логику конфиг файлом статически прописать, чтобы в GitHub его пушить, а Алиса подтягивала?

**Comment by abitrolly** [comment]
Датчик https://airmq.by/

**Comment by andrei-podholms** [comment]
> А что за датчик? Может имеет смысл просто его показания или событие в Алису кидать (http, mqtt и так далее), а там уже прописать логику, что с этим делать.

можно, в каком формате куда кидать?

**Comment by sergejey** [comment]
В MQTT публиковать можно? На Алисе брокер крутится, можно туда любым топиком значение (одно или JSON-ом пачку), а она подхватит.

**Comment by andrei-podholms** [comment]
Да, я прокси сервис буду на raspberry pi подымать, поэтому могу конвертировать во что угодно. Только формат бы узнать.
Сейчас меряется 3 вида пыли, влажность, температура, дваление. Можно кидатьнотификацию когда нормы превышаются.

**Comment by andrei-podholms** [comment]
@sergejey а можешь дать ссылочку в коде где кидают подобное значение на алису? Я тогда вкручу его в https://github.com/andrei-podholms/airnotifyer

**Comment by sergejey** [comment]
а, слушай, так я смотрю оно просто по json выдаёт данные при http запросе. просто скажи тогда ip-шник сенсора в сети и я прикручу остальное.

**Comment by andrei-podholms** [comment]
http://g7.db3gw.keenetic.pro/d/s2gwd138Mz/mhs?orgId=1&from=now-7d&to=now

user: public
pass: public

IP в ХСе 192.168.128.65

**Comment by andrei-podholms** [comment]
> а, слушай, так я смотрю оно просто по json выдаёт данные при http запросе. просто скажи тогда ip-шник сенсора в сети и я прикручу остальное.

Я допишу парсилку. Только там еще вопрос был в том чтобы описать добавление новых сервисов к общей системе.
